tests/l2/README.md

```diff
@@ -67,6 +68,38 @@ $ oc logs intel-dgpu-clinfo-56mh2
   Device OpenCL C Version                         OpenCL C 1.2
   Device OpenCL C all versions                    OpenCL 
 ```                                               
+#### hwinfo
+
+This workload runs ```hwinfo``` utilizing the i915 resource from GPU provisioning and displays the related GPU information. Refer to [link](https://dgpu-docs.intel.com/driver/installation.html#verify-install)
+
+
+*	Build the workload container image. 
+
+```$ oc apply -f https://raw.githubusercontent.com/intel/intel-technology-enabling-for-openshift/main/tests/l2/dgpu/hwinfo_build.yaml ```
+
+*	Deploy and execute the workload.
+
+```$ oc apply -f https://raw.githubusercontent.com/intel/intel-technology-enabling-for-openshift/main/tests/l2/dgpu/hwinfo_job.yaml```
+
+* Check the results
+``` 
+  $ oc get pods
+  intel-dgpu-hwinfo-1-build   0/1     Completed   0          111m
+  intel-dgpu-hwinfo-l4572     0/1     Completed   0          110m
+```
+```
+$ oc logs intel-dgpu-hwinfo-l4572  
+  88: PCI 3700.0: 0300 VGA compatible controller (VGA)
+  [Created at pci.386]
+  Unique ID: ug9x.K5dXxUXtHX2
+  Parent ID: QGIh.mr2N3fBJq5F
+  SysFS ID: /devices/pci0000:30/0000:30:02.0/0000:31:00.0/0000:32:00.0/0000:33:00.0/0000:34:08.0/0000:35:00.0/0000:36:01.0/0000:37:00.0
+  SysFS BusID: 0000:37:00.0
+  Hardware Class: graphics card
+  Model: "Intel VGA compatible controller"
+  Vendor: pci 0x8086 "Intel Corporation"
+  Device: pci 0x56c1
+```                        
```

Comment:
What we need is the i915 driver information shown by hwinfo like
  
Driver: "i915"
  Driver Modules: "i915"
  Memory Range: 0x23fe7e000000-0x23fe7fffffff (ro,non-prefetchable)
  Memory Range: 0x236000000000-0x237fffffffff (ro,non-prefetchable)
  IRQ: 138 (447 events)
  Module Alias: "pci:v00008086d00000BD5sv00008086sd00000000bc03sc80i00"
  Driver Info #0:
    Driver Status: i915 is active
    Driver Activation Cmd: "modprobe i915"
  Config Status: cfg=new, avail=yes, need=no, active=unknown
  Attached to: #26 (PCI bridge)

Should we provision the driver and then run the hwinfo?

Reply:
yes we should provision driver. this can be good l1 test case.

Reply:
updated and added complete output